src/test/java/com/iexec/sms/tee/session/scone/SconeSessionMakerServiceTests.java

```diff
@@ -72,13 +71,12 @@ void beforeEach() {
     // region getSessionYml
     @Test
     void shouldGetSessionYml() throws Exception {
-        TeeEnclaveConfiguration enclaveConfig = mock(TeeEnclaveConfiguration.class);
+        TeeEnclaveConfiguration enclaveConfig = TeeEnclaveConfiguration.builder()
+                .fingerprint(APP_FINGERPRINT).entrypoint(APP_ENTRYPOINT).build();
```

Comment:
I find it clearer on multiple lines.
```suggestion
        TeeEnclaveConfiguration enclaveConfig = TeeEnclaveConfiguration.builder()
                .fingerprint(APP_FINGERPRINT)
                .entrypoint(APP_ENTRYPOINT)
                .build();
```

Reply:
done in c1d5287